src/project/project.service.ts

```diff
@@ -561,12 +563,17 @@ export class ProjectService {
       });
       switch (enrollment?.requestState) {
         case RequestState.Pending:
+          if (enrollment.sender !== null && enrollment.sender !== undefined) {
+            throw new BadRequest(
+              'Este usuario ya ha sido invitado para inscribirse en este proyecto',
+            );
+          }
```

Comment:
I believe this validation will never enter as per our logic, an admin won't see invitation button if user has been already invited. Having said that, we could keep it, double verification from backend is always good